<?xml version="1.0" ?>
<launch>
	<node pkg="reconfigure" name="cheetah_cur_reconfigure" type="cur_reconfigure_server.py" output="screen" />

	<node name="rqt_gui" pkg="rqt_gui" type="rqt_gui" args="-s reconfigure"/>

	<node pkg="reconfigure" name="cheetah_cur_pub_rqt" type="cur_pub.py" output="screen"/>
</launch>